vm/systemSmartContracts/staking.go

```diff
@@ -1196,6 +1198,26 @@ func (s *stakingSC) switchJailedWithWaiting(args *vmcommon.ContractCallInput) vm
 	return vmcommon.Ok
 }
 
+func (s *stakingSC) tryRemoveJailedNodeFromStaked(registrationData *StakedDataV2_0) {
+	if !s.flagCorrectJailedNotUnstakedEmptyQueue.IsSet() {
+		s.removeAndSetUnstaked(registrationData)
+	} else {
```

Comment:
add a return here. and there is no need for else

Reply:
fixed